runnables/extract_ts_features.py

```diff
@@ -2,7 +2,7 @@
 import os
 import pandas as pd 
 from sqlalchemy import create_engine
-from ts_feature_extractor import add_features_extracted_from_ts_data
+from utils.ts_feature_extractor import add_features_extracted_from_ts_data
```

Comment:
We normally do not add a `utils` package since the name `utils` do not refer to a specific functionality which may cause the code difficult to read.

Reply:
Good idea. Naming it time_series_processing will be more explicit.